README.md

```diff
@@ -2,3 +2,27 @@
 
 This project contains scripts that are consumed by load-runner, for
  load-testing the [demo cloud app](https://github.com/feedhenry-raincatcher/raincatcher-demo-cloud).
+
+## Setup
+
+There's a setup script at `./setup/usersWorkorders.js` which creates a
+ specified number of users, and creates a workorder for each one. Running it with node on the command line will show details of the parameters that it expects:
+
+``` bash
+node setup/usersWorkorders.js
```

Comment:
you could `chmod +x` this and add a `#!/usr/bin/env node` shebang, and maybe also `npm run setup`